docs/expression_syntax.md

```diff
@@ -170,7 +170,13 @@ will extract `message` field from the event and compares it's length with 20. Th
 Currently the following functions are supported
  * `length()`
    - takes one argument of JsonPointer type
-   - returns the length of the value of the argument passed if it's type is string. For example, `length("/message")` returns 10 if the key `message` exists in the event and has the value of `"1234567890"`.
+   - returns the length of the value of the argument passed if it's type is string.
+   For example, `length("/message")` returns 10 if the key `message` exists in the event and has the value of `"1234567890"`.
+ * `hasTags()`
+   - takes at least one argument
+   - all arguments must be of String type
+   - returns true if all arguments are present in the event's tags, returns false otherwise
+   For example, if event has tags "tag1", "tag2", and "tag3", `hasTags("tag1")` or `hasTags("tag1", "tag2")` would return true and `hasTags("tag4")` would return false.
```

Comment:
What about `hasTags("tag1", "tag4")`? False? Please add this as an example as well